application/src/main/java/org/togetherjava/tjbot/features/chaptgpt/ChatGptService.java

```diff
@@ -59,15 +76,18 @@ public Optional<String> ask(String question) {
                 .model("gpt-3.5-turbo")
                 .messages(List.of(chatMessage))
                 .frequencyPenalty(0.5)
-                .temperature(0.7)
+                .temperature(0.3)
                 .maxTokens(MAX_TOKENS)
                 .n(1)
                 .build();
-            return Optional.ofNullable(openAiService.createChatCompletion(chatCompletionRequest)
+
+            String response = openAiService.createChatCompletion(chatCompletionRequest)
                 .getChoices()
                 .get(0)
                 .getMessage()
-                .getContent());
+                .getContent();
+
+            return AIResponseParser.parse(response);
```

Comment:
no, u can't just remove the optional from there and return it in ur parse method

Reply:
Not sure what is going on here. Do you mean the function call? It originally returned an optional string array. I now have line 90 as:
```java
return Optional.of(AIResponseParser.parse(response));
```
Is this what you mean?

Reply:
no, go by the logic, previously Optional.ofNullable() was wrapped on the return statement, which now u store in ur `response` variable. So that means it can be null. So u'll have to handle that case in either ur parse method but i'd recommend just return an empty optional if that's the case and not invoke the parse method at all

Reply:
I just pushed what I think is the change you are requesting. This is now outdated.